### PR TITLE
Tweak some README.md structure

### PR DESCRIPTION
**Pull request by kdeldycke**
* Use sharp-leading subtitles (no need to maintain the underline at length)
* Use a period to end description (description from cookiecutter template is dot-less)
* Invite user to fill in the blank of the `README` with usage example and previous plugin `README`

### CODE REVIEW

File: {{cookiecutter.repo_name}}/{{cookiecutter.readme}} [hunk 1]
```diff
@@ -4,17 +4,22 @@
 [![PyPI Version](https://img.shields.io/pypi/v/{{ cookiecutter.distribution_name }})](https://pypi.org/project/{{ cookiecutter.distribution_name }}/)
 ![License](https://img.shields.io/pypi/l/{{ cookiecutter.distribution_name }}?color=blue)
 
-{{ cookiecutter.description }}
+{{ cookiecutter.description }}.
```
Comment (justinmayer) [inline {{cookiecutter.repo_name}}/{{cookiecutter.readme}}]: What happens when someone includes a period in the `description` field when answering the interactive questions? Won't that (unexpectedly) result in two periods? If so, perhaps the "principle of least astonishment" approach is to let the user handle the period?

File: {{cookiecutter.repo_name}}/{{cookiecutter.readme}} [hunk 2]
```diff
@@ -4,17 +4,22 @@
...
 
-Installation
-------------
+
+## Installation
```
Comment (justinmayer) [inline {{cookiecutter.repo_name}}/{{cookiecutter.readme}}]: Regarding the Markdown heading style, I spend a **lot** of time looking at plain-text Markdown (i.e., not rendered), and I usually find the underlined headers _much_ more "scannable" and readable than `#` / `##`. To me, that readability is worth the prospect of potentially having to someday adjust the underline length, although in reality I don't think headings change often enough for that to be a significant concern. So… while not a huge deal either way, I'm not sure I totally dig this proposed change.

File: {{cookiecutter.repo_name}}/{{cookiecutter.readme}} [hunk 3]
```diff
@@ -4,17 +4,22 @@
...
 This plugin can be installed via:
 
     python -m pip install {{ cookiecutter.distribution_name }}
 
-Contributing
-------------
+
+## Usage
+
+<<Add plugin details here>>
```
Comment (justinmayer) [inline {{cookiecutter.repo_name}}/{{cookiecutter.readme}}:20]: Love the idea of adding a blank section to encourage folks to add more details. Great! 😁

**Review by justinmayer — COMMENTED** [review]
Made a few comments. Let me know what you think.

File: {{cookiecutter.repo_name}}/{{cookiecutter.readme}} (hunk 1 above)
Comment (kdeldycke) [reply {{cookiecutter.repo_name}}/{{cookiecutter.readme}}]: Aha OK, I'll remove it. It was just nitpicking anyway! 😁

File: {{cookiecutter.repo_name}}/{{cookiecutter.readme}} (hunk 2 above)
Comment (kdeldycke) [reply {{cookiecutter.repo_name}}/{{cookiecutter.readme}}]: Again, I've no issue reverted those proposed changes. I'll followed your preferences ! :)

**Comment by kdeldycke** [comment]
All fixed up! 🙂

**Comment by justinmayer** [comment]
Thanks, Kevin!

**Pull request merged**